paderbox/io/csv_module.py

```diff
@@ -66,6 +66,9 @@ def load_csv(
     [{'a': '1', 'b': '2', 'c': '3'}, {'a': '4', 'b': '5', 'c': '6'}]
     >>> loads_csv(content, fieldnames=['d', 'e', 'f'], sniffer=True)
     [{'d': 'a', 'e': 'b', 'f': 'c'}, {'d': '1', 'e': '2', 'f': '3'}, {'d': '4', 'e': '5', 'f': '6'}]
+    >>> content = 'a\\tb\\tc\\n1\\t2\\t3\\n4\\t5\\t6'
```

Comment:
Are those double back-slashes correct?

Reply:
Yes, they are correct. The docstring is interpreted from python and you have to escape some characters.
I tested if I can use `r'...'`. It worked, so I removed the double back-slash,